api/environments/sdk/serializers_mixins.py

```diff
@@ -4,5 +4,5 @@ def to_representation(self, instance):
         environment = self.context["request"].environment
         if environment.hide_sensitive_data:
             for field in self.sensitive_fields:
-                data.pop(field)
+                data[field] = [] if isinstance(data[field], list) else None
```

Comment:
Is `None` an acceptable value for the feature description field? It should ideally be an empty string, right?